src/tor_relay_scanner/scanner.py

```diff
@@ -164,4 +166,9 @@ async def main_async():
 
 
 def main():
-    asyncio.run(main_async())
+    parser = argparse.ArgumentParser(description=DESCRIPTION)
+    parser.add_argument('-n', type=int, dest='num_relays', nargs='?', default=30, help='Choose the number of relays to test.')
+    parser.add_argument('-g', '--goal', type=int, dest='working_relay_num_goal', nargs='?', default=5, help='Choose the max number of relays to scan.')
+    parser.add_argument('--timeout', type=float, nargs='?', default=10.0)
```

Comment:
Please write a text description for this option

Reply:
num_relays → The number of concurrent relays tested
working_relay_num_goal → Test until at least this number of working relays are found
timeout → socket connection timeout